core/services/relay/evm/functions/contract_transmitter.go

```diff
@@ -87,18 +110,54 @@ func NewFunctionsContractTransmitter(
 		return nil, err
 	}
 	return &contractTransmitter{
-		contractABI:         contractABI,
-		transmitter:         transmitter,
-		transmittedEventSig: transmitted.ID,
-		lp:                  lp,
-		contractReader:      caller,
-		lggr:                lggr.Named("OCRContractTransmitter"),
-		reportToEvmTxMeta:   reportToEvmTxMeta,
-		contractVersion:     contractVersion,
-		reportCodec:         codec,
+		contractABI:                 contractABI,
+		transmittedEventSig:         transmitted.ID,
+		lp:                          lp,
+		contractReader:              caller,
+		lggr:                        lggr.Named("OCRContractTransmitter"),
```

Comment:
would it help when debugging if the lggr was named: `OCRFunctionsContractTransmitter`?